src/Common/tests/System/Net/Prerequisites/Servers/CoreFxNetCloudService/WebServer/WebSocket/EchoWebSocket.ashx.cs

```diff
@@ -15,12 +15,15 @@ namespace WebServer
     public class EchoWebSocket : IHttpHandler
     {
         private const int MaxBufferSize = 128 * 1024;
+        private bool _replyWithPartialMessages;
```

Comment:
I didn't find any mention of initialization in the coding guidelines document:
https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md

We know that C# initializes fields in a class (but not stack variables).  But I think the compiler would remove the redundant code anyways.  I just think initializing it makes the code easier to read and understand.